components/compositing/compositor.rs

```diff
@@ -1222,6 +1227,10 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
                 rect_changed = rect != webview.rect;
                 size_changed = rect.size() != webview.rect.size();
                 webview.rect = rect;
+
+                if let Some(r) = radius {
+                    webview.radius = r;
+                }
```

Comment:
```suggestion
radius.map(|r| webview.radius = r);
```

Reply:
I actually prefer `if let` here because we don't use the return value of the `map`.